modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -184,6 +184,26 @@ public void testAddCustomObjectField() throws Exception {
 						true
 					).build())));
 
+		// Localized object fields must not be required
+
+		_assertFailure(
```

Comment:
Hey @seltong,
I remember that @murilostodolni did a refactor in all tests to remove this private method `_assertFailure`. Brian asked him to move this private method to` AssertUtils` class because this method has been used a lot. 

Also, I tried to search for this private method in your PR and I did not find. Are you sure that this is compiling? Because I saw that the other usages of this method has been replaced in this class and this private method does not exists anymore.

Reply:
You are right @marcelabc.

@seltong you can used the `AssertUtils.assertFailure`

Reply:
I will check this, thanks